container.go

```diff
@@ -56,32 +55,14 @@ const (
 // Supported resource types are Network and Request Types are Add/Remove
 type ResourceModificationRequestResponse = schema1.ResourceModificationRequestResponse
 
-// createContainerAdditionalJSON is read from the environment at initialisation
-// time. It allows an environment variable to define additional JSON which
-// is merged in the CreateContainer call to HCS.
-var createContainerAdditionalJSON string
-
-func init() {
-	createContainerAdditionalJSON = os.Getenv("HCSSHIM_CREATECONTAINER_ADDITIONALJSON")
-}
 
 type container struct {
 	system *hcs.System
 }
 
 // CreateContainer creates a new container with the given configuration but does not start it.
 func CreateContainer(id string, c *ContainerConfig) (Container, error) {
-	system, err := hcs.CreateContainer(id, c)
-	if err != nil {
-		return nil, err
-	}
-	return &container{system}, err
-}
-
-// CreateContainerWithJSON creates a new container with the given configuration but does not start it.
-// It is identical to CreateContainer except that optional additional JSON can be merged before passing to HCS.
-func CreateContainerWithJSON(id string, c *ContainerConfig, additionalJSON string) (Container, error) {
```

Comment:
Are you certain no one is using this? This is a breaking change.

Reply:
99.9%. When I added it, I thought docker would be able to use it, but instead it's only done through an environment variable. It's extremely unlikely a 3rd party would use this as we didn't document the v1 schema.

Reply:
I didn't find any references in GitHub code search. So I'm OK with this.